go/test/endtoend/vtgate/system_schema_test.go

```diff
@@ -207,12 +212,9 @@ func TestMultipleSchemaPredicates(t *testing.T) {
 	require.EqualValues(t, 4, len(qr1.Fields))
 
 	// test a query with two keyspace names
-	query = fmt.Sprintf("select t.table_schema,t.table_name,c.column_name,c.column_type "+
```

Comment:
please keep the original test since it is testing different path, just assert the equal values of 2.